### PR TITLE
[prometheus-pgbouncer-exporter] Add chart for pgbouncer-exporter

### PR DESCRIPTION
**Pull request by Pluggi**
#### What this PR does / why we need it:

Adds a chart for pgbouncer_exporter.

#### Special notes for your reviewer:

I am not sure if the way I pass the connection string is okay?

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by Pluggi** [comment]
@monotek could you please take a look?

**Review by monotek — CHANGES_REQUESTED** [review]
Please Fix CI.

@prometheus-community/helm-charts-admins 
please review too, as we need codeowners changed.

File: charts/prometheus-pgbouncer-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+sources:
+  - https://github.com/prometheus-community/pgbouncer_exporter
+
+keywords:
+  - prometheus
+  - pgbouncer
+
+maintainers:
+  - name: Pluggi
+    email: pluggi512@gmail.com
```
Comment (zanhsieh) [inline charts/prometheus-pgbouncer-exporter/Chart.yaml:14]: I think we need more than one maintainer. I (zanhsieh, zanhsieh@gmail.com) could help a little bit.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/prometheus-pgbouncer-exporter/Chart.yaml (hunk 1 above)
Comment (Pluggi) [reply charts/prometheus-pgbouncer-exporter/Chart.yaml:14]: I added you @zanhsieh :)
Thank you

**Comment by monotek** [comment]
@torstenwalter or @scottrigby 
Could one of you review / approve please?

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@monotek @torstenwalter 
Do you think add this chart and maintainers to https://github.com/prometheus-community/helm-charts/blob/main/.github/CODEOWNERS would help?

**Review by monotek — CHANGES_REQUESTED** [review]
> @monotek @torstenwalter Do you think add this chart and maintainers to https://github.com/prometheus-community/helm-charts/blob/main/.github/CODEOWNERS would help?

It's already added in this pr: https://github.com/prometheus-community/helm-charts/blob/33c132a25ab5937b884f5d8ea51a5fa3c9173501/.github/CODEOWNERS#L25

Imho the chart itself needs a fix. 
If the container expects a running instance of pgbouncer we might need to add one for the chart test ci.
We did something like this for the kafka exporter for example: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-kafka-exporter/values.yaml#L132-L133

**Comment by zanhsieh** [comment]
@Pluggi 
We probably need to get postgresql and pgbouncer chart ready on Chart.yaml > dependencies, e.g.
```
# Chart.yaml
...
dependencies:
- name: postgresql
  version: "10.13.2"
  repository: https://charts.bitnami.com/bitnami
  condition: postgresql.enabled

# values.html
...
# If enabled Postgresql dependency chart will be used.
# This is only needed for the CI job so it should always be disabled.
postgresql:
  enabled: false
```

**Review by monotek — COMMENTED** [review]
> Dunno if that is sufficient enough.

Check Kafka's ci directory. There the dependencies are enabled.

**Pull request closed without merging**